fdbclient/BlobWorkerCommon.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * BlobWorkerCommon.h
+ *
```

Comment:
Oh i didn't realize you added this in a separate file.
We already have BlobGranuleCommon.h, it probably makes more sense to move it there.
If you don't move it you should add this file to fdbclient/CMakeLists.txt though

Reply:
I initially added it there. But unfortunately, adding it there causes a name conflict since `BlobGranuleCommons` is already included as part of `BlobManager` and `BlobManager` has a struct called `BlobWorkerStats` (which I added earlier this week) :/

Also, it didn't feel like it belonged in `BlobGranuleCommons` when I looked at the other structs in that file.
I'll go ahead and add it to `fdbclient/CMakeLists.txt` if these are sufficient reasons to keep it separate.